packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -523,6 +592,27 @@ export default decorate([
           [name]: value,
         })),
       }),
+      addReportReceiver({ setGlobalSettings }, value) {
+        const { propSettings, settings = propSettings } = this.state
+        const reportReceivers = defined(
+          settings.getIn(['', 'reportReceivers']),
+          []
+        )
+        if (!reportReceivers.some(receiver => receiver === value)) {
+          setGlobalSettings({
+            name: 'reportReceivers',
+            value: (reportReceivers.push(value), reportReceivers),
```

Comment:
Shouldn't it be immutable? Or maybe `setGlobalSettings` takes care of it?

Reply:
`settings.getIn` returns a new object because we use `immutable`.